go-controller/pkg/ovn/master.go

```diff
@@ -394,6 +395,12 @@ func (oc *Controller) SetupMaster(masterNodeName string) error {
 		return err
 	}
 
+	err = acl.PurgeRejectRules(oc.clusterPortGroupUUID)
```

Comment:
should we be doing this in the services sync functions instead?

Reply:
I think we don't need this overhead every time we sync a service, doing at start is enough

Reply:
sorry I think the confusion is from the overloaded term now of "sync". So in the old controllers we have a sync function that runs when the watcher spins up, it is supposed to handle removing all old stuff from OVN. In the services watcher sync method there you would call this. In the new services_controller, this is called as part of the "repair"

Reply:
Oh ok, I'll move then.